CRM/Sepa/Upgrader.php

```diff
@@ -21,7 +21,7 @@
 /**
  * Collection of upgrade steps.
  */
-class CRM_Sepa_Upgrader extends CRM_Sepa_Upgrader_Base {
```

Comment:
Not sure if this change is compatible with CiviCRM 5.45. I found this change breaking in older installations because it expects the base class to be present in CiviCRM core. Which is a wrong assumption if the extension needs to be compatible with older CiviCRM versions.

Reply:
The core class `CRM_Extension_Upgrader_Base` was added in CiviCRM `5.38`, so this should be fine. *Civix* is checking that and adjusting the `info.xml` accordingly.

Reply:
Ok thanks! I did not know that